doc/source/dev/index.rst

```diff
@@ -2,15 +2,323 @@
+
+   * If you want, post on the `mailing list
+     <https://mail.python.org/mailman/listinfo/numpy-deve>`_ to explain your changes or
+     to ask for review.
+
+For a more detailed discussion, read the documents at the bottom of this page.
+
+4. Review process:
+
+   * Reviewers (the other developers and interested community members) will
+     write inline and/or general comments on your Pull Request (PR) to help
+     you improve its implementation, documentation and style.  Every single
+     developer working on the project has their code reviewed, and we've come
+     to see it as friendly conversation from which we all learn and the
+     overall code quality benefits.  Therefore, please don't let the review
+     discourage you from contributing: its only aim is to improve the quality
+     of project, not to criticize (we are, after all, very grateful for the
+     time you're donating!).
+
+   * To update your PR, make your changes on your local repository
+     and commit. As soon as those changes are pushed up (to the same branch as
+     before) the PR will update automatically. You can subscribe to this PR to
+     get an e-mail update whenever there are changes.
+
+   * Various continuous integration (CI) services are triggered after each PR
+     update to build the code, run unit tests, measure code coverage and check
+     coding style of your branch. The CI tests must pass before your PR can be
+     merged. If CI fails, you can find out why by clicking on the "failed"
+     icon (red cross) and inspecting the build and test log.
+
+   * A code-change PR must be **approved** by at least two core team members
+     before merging. Approval means the core team member has carefully reviewed
+     the changes, and the PR is ready for merging.
+
+5. Document changes
+
+   If your change introduces any user-facing modifications, please update the
+   current release notice under ``doc/release/X.XX-notes.rst``
+
+   If your change introduces a deprecation, make sure to follow the
+   `deprecation policy <http://www.numpy.org/neps/nep-0023-backwards-
+    compatibility.html>`_.
+
+.. note::
+
+   To reviewers: if it is not obvious from the PR description, add a short
+   explanation of what a branch did to the merge message and, if closing a
+   bug, also add "Closes #123" where 123 is the issue number.
+
+
+Divergence between ``upstream master`` and your feature branch
+--------------------------------------------------------------
+
+If GitHub indicates that the branch of your Pull Request can no longer
+be merged automatically, merge the master branch into yours, you must
+resolve the conflict. One way to do this is to `rebase on master <rebasing-on-
+master>`.
+
+Guidelines
+----------
+
+* All code should have tests (see `test coverage`_ below for more details).
+* All code should be `documented <docstring-standard>`.
+* No changes are ever committed without review and approval by two core
+  team members.  Ask on the
```

Comment:
I've seen / been responsible for a handful of merges with just one reviewer. Are we making this against policy from now on, or sticking with a "use judgement" based approach where 2 is the norm, but trivial fixes are fine with just one?

Reply:
No, this is just a copy-paste error - one reviewer is perfectly fine

Reply:
scikit-image has always had a different policy

Reply:
fixing